readme.md

```diff
@@ -145,6 +145,10 @@ Check if `value` (number) is in the range of `0` to `upperBound`.
 is.inRange(3, 10);
 ```
 
+##### .domElement(value)
```

Comment:
Maybe `.domNode`? Just a question.

Reply:
There are 7 types of nodes supported currently .
Reference : https://developer.mozilla.org/en-US/docs/Web/API/Node/nodeType

Element is one of them. So I feel .domElement() is more appropriate for this pull request

However I create a different function .domNode() which will check if value is a node?

Reply:
> Element is one of them. So I feel .domElement() is more appropriate for this pull request

Makes sense.

> However I can create a different function .domNode() which will check if value is a node or not?

I don't think its use case is that common, but I'll let @sindresorhus decide if he wants it.

Reply:
> I don't think its use case is that common, but I'll let @sindresorhus decide if he wants it.

Nah, only interested in `domElement`.